### PR TITLE
update README.md to mention OpenCL conformance

### PR DESCRIPTION
**Pull request by rjodinchr**

### CODE REVIEW

**Comment by rjodinchr** [comment]
This PR is also updating `docs/OpenCLCOnVulkan.md`

**Review by alan-baker — APPROVED** [review]

File: docs/OpenCLCOnVulkan.md [hunk 1]
```diff
@@ -887,51 +872,37 @@ These extension built-in functions are not supported:
...
-`ceil`, `cos`, `cosh`, `exp`, `exp2`, `exp10`, `fabs`, `floor`, `fmax`, `fmin`,
-`log`, `log2`, `log10`, `mad`, `pow`, `powr`, `rint`, `rsqrt`, `sin`, `sinh`,
-`tan`, `trunc`, `half_cos`, `half_exp`, `half_exp2`, `half_exp10`, `half_log`,
-`half_log2`, `half_log10`, `half_powr`, `half_rsqrt`, `half_sin`, `half_tan`,
-`clamp`, `degrees`, `mix`, `radians`, `sign`, `smoothstep`, `step`, `cross`,
-`dot`, `normalize`, `fast_distance`, `fast_length`, `fast_normalize`.
+Find the latest list in clspv's libclc [SOURCES file](https://github.com/llvm/llvm-project/blob/main/libclc/clspv/lib/SOURCES).
 
-#### Implementations Using Builtin Library
+Here is the list as of March 2024:
```
Comment (lpavank) [inline docs/OpenCLCOnVulkan.md:884]: Nit/Question: Trying to understand, given there are multiple HW implementations with different caps (and/or use CLVK flags to CLSPV to tell which built-ins they want native vs libclc). Is this list "lowest common denominator" or all those? Trying to understand, how we got this list (given various HW implementations had different caps or its just the two certified).

Comment (rjodinchr) [reply docs/OpenCLCOnVulkan.md:884]: It's just the list of builtins coming from libclc. It's coming from the libclc SOURCES file (the link is above in the PR).

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # clspv [![Discord Shield](https://discordapp.com/api/guilds/1002628585250631681/widget.png?style=shield)](https://discord.gg/xsVdjmhFM9)
 
-**Clspv** is a prototype compiler for a subset of OpenCL C to Vulkan
-compute shaders.
+**Clspv** is a compiler for OpenCL C targeting Vulkan compute shaders.
```
Comment (dneto0) [inline README.md:3]: Congratulations!

**Review by dneto0 — COMMENTED** [review]
Approved!

Congratulations, this is a great milestone.

**Pull request merged**